dbeam-core/src/main/java/com/spotify/dbeam/options/JdbcExportArgsFactory.java

```diff
@@ -84,6 +88,15 @@ public static QueryBuilderArgs createQueryArgs(JdbcExportPipelineOptions options
         .build();
   }
 
+  private static Optional<String> resolveSqlQueryParameter(JdbcExportPipelineOptions options)
+      throws IOException {
+    if (options.getSqlFile() != null) {
+      return Optional.ofNullable(new ParameterFileReader().readAsResource(options.getSqlFile()));
```

Comment:
Do we need `Optional.ofNullable` here? When does `readAsResource` return null? I think it should be reported to the user instead of the exception getting swallowed.

Reply:
Addressed.